pql/QueryProcessor.py

```diff
@@ -50,6 +51,7 @@ def match(self, token: str) -> None:
 
     def error(self, info: str) -> None:
         log.error("ERROR: " + info)
+        sys.exit("#ERROR " + info)
```

Comment:
To tutaj zapewnia poprawne działanie?

Reply:
Zamykamy tutaj program, to kolejne zapytanie w pipetesterze się wywoła?

Reply:
wywal te sys.exit i zamiast tego wpisz:
raise Exception()

w main_pql.py zrób tak:
w mainie masz metodę query_processor.generate_query_tree(query)

To ją otocz tryem, złap wyjątem i zwróć "#ERROR"
Czyli coś takiego:
try:
    query_processor.generate_query_tree(query)
except:
    return "#ERROR - jakieś ewentualne info o wyjątku itd..."